core/lb.c

```diff
@@ -929,6 +933,9 @@ void
 oio_lb_world__feed_slot (struct oio_lb_world_s *self, const char *name,
 		const struct oio_lb_item_s *item)
 {
+	if (!item || item->weight <= 0)
+		return;
```

Comment:
Change reverted. Idem for metautils/lib/lb.c. Juste store zeroed services and ignore them.